.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: shellcheck
+on:
+  pull_request:
+    paths:
+      - "**/**.sh"
+
+jobs:
+  shellcheck:
+    name: shellcheck
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Run ShellCheck
+        uses: bewuethr/shellcheck-action@cdeea6bdd72272a5b839e30edd7efa3f83c424cf
```

Comment:
let's add a new line here 
Context: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline